### ISSUE TITLE
Add scale option to save_png 

### ISSUE THREAD

**Issue by andymeneely** [andymeneely/squib#183]
It would be nice to scale your output to a different size. For example, figures for rules sheets don't need to be at full resolution.

This could be done by adding a "width/height" to `save_png`, or simply adding a `scale` option. Width/height would be easier to work with for users, I think, and would be more consistent with everything else.

**Comment by andymeneely** [comment]
I  had a hard time implementing this one. It's trickier than it sounds.

**Comment by sparr** [comment]
Got any guidance on what you did or didn't accomplish in 2afb2a6?

**Comment by andymeneely** [comment]
Getting rotating and trim working properly was the tricky part.

**Comment by msoucy** [comment]
Is there any chance of this being revisited in the future? I'd love to get rid of my hacks for using magick to make it work. With tools like screen top and tabletop Playground, the default size of 2.5x3.5in cards put 10 in a row is too large to upload.

**Comment by andymeneely** [comment]
Yeah I can take another look at it. For now what I do to work around it is have a Squib deck where the _input_ is from the _output folder. I found that easier than going to other tools.
